ruma-events/src/key/verification/start.rs

```diff
@@ -106,8 +107,14 @@ impl MSasV1Content {
     /// `MessageAuthenticationCode::HkdfHmacSha256`.
     /// * `short_authentication_string` does not include `ShortAuthenticationString::Decimal`.
     pub fn new(options: MSasV1ContentOptions) -> Result<Self, InvalidInput> {
-        if !options.key_agreement_protocols.contains(&KeyAgreementProtocol::Curve25519) {
-            return Err(InvalidInput("`key_agreement_protocols` must contain at least `KeyAgreementProtocol::Curve25519`".into()));
+        if !options.key_agreement_protocols.contains(&KeyAgreementProtocol::Curve25519)
+            && !options
+                .key_agreement_protocols
+                .contains(&KeyAgreementProtocol::Curve25519HkdfSha256)
+        {
+            return Err(InvalidInput("`key_agreement_protocols` must contain at \
+                                    least `KeyAgreementProtocol::Curve25519` or \
+                                    `KeyAgreementProtocol::Curve25519HkdfSha256`".into()));
```

Comment:
I am surprised rustfmt doesn't complain about this. Can you make it use block indentation since generally the standard formatting for Rust is *never* visual indentation?

So probably

```suggestion
            return Err(InvalidInput(
                "`key_agreement_protocols` must contain at least \
                `KeyAgreementProtocol::Curve25519` or \
                `KeyAgreementProtocol::Curve25519HkdfSha256`".into(),
            ));
```

(but maybe if you write that rustfmt changes something up again)

Same for the two error messages below.

Reply:
It did complain and I ran `cargo fmt`, or am I missing something about this? Github claims that the suggestion is outdated so it's probably fine now.

Reply:
Oh, seems GitHub just showed me an outdated diff :facepalm: